tests/integration/test_api/test_config/test_limit_eps/test_limit_eps.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+from pathlib import Path
+import requests
+
+from . import CONFIGURATIONS_FOLDER_PATH, TEST_CASES_FOLDER_PATH
+from wazuh_testing.constants.api import CONFIGURATION_TYPES, MANAGER_CONFIGURATION_ROUTE
+from wazuh_testing.constants.daemons import API_DAEMONS_REQUIREMENTS
+from wazuh_testing.modules.api.utils import login, get_base_url
+from wazuh_testing.utils.configuration import get_test_cases_data, load_configuration_template, get_wazuh_conf
+
+# Marks
+pytestmark = pytest.mark.server
+
+# Variables
+# Used by add_configuration to select the target configuration file
+configuration_type = CONFIGURATION_TYPES[0]
+
+# Paths
+test_configuration_path = Path(CONFIGURATIONS_FOLDER_PATH, 'limit_eps_api_config.yaml')
+test_cases_path = Path(TEST_CASES_FOLDER_PATH, 'limit_eps_config.yaml')
```

Comment:
Again, not sure if this is a must. But all the other tests configurations and cases start with `configuration` and `cases` respectively. We should check whether this is necessary and apply the adjustments.

```suggestion
test_configuration_path = Path(CONFIGURATIONS_FOLDER_PATH, 'configuration_limit_eps_configuration.yaml')
test_cases_path = Path(TEST_CASES_FOLDER_PATH, 'cases_limit_eps_configuration.yaml')
```

Reply:
Changed in commit 55c3deb54b1205546430149e7a80b348833da665.